RELEASE_NOTES.md

```diff
@@ -0,0 +1,5 @@
+#### 1.0.1 May 27 2016 ####
+Removed non-generic IPublisher and ISubscriber interfaces.
```

Comment:
Hi Chris,

Thanks for the prompt response!

If nobody is using 1.0.0 and 1.0.1, I suggest talking the opportunity to
unlist them:
http://stackoverflow.com/questions/11654652/how-to-remove-nuget-package-from-server

Would starting at 1.0.0-milestone1 and then work towards 1.0.0 make sense?

I *strongly* recommend against making any non-beta/non-milestone releases
until feature parity with the -jvm version.
Our (-jvm) final versions are always handled as Issues that everyone sign
off on before they are published.
(For tasks which are difficult to reverse, asking for input/feedback
beforehand is always simpler and cheaper, or so I have found.)

Please don't hesitate in pinging me either here or via email if you have
questions.

-- 
Cheers,
√
On May 29, 2016 18:11, "Chris Constantin" <notifications@github.com> wrote:

> In RELEASE_NOTES.md
> <https://github.com/reactive-streams/reactive-streams-dotnet/pull/7#discussion_r65007389>
> :
>
> > @@ -0,0 +1,5 @@
> > +#### 1.0.1 May 27 2016 ####
> > +Removed non-generic IPublisher and ISubscriber interfaces.
>
> Hi Victor, it was my mistake in not publishing the initial release as beta.
> For now there are no public libraries referencing Reactive.Streams. We
> just merged akka-streams into the main branch, but it won't be
> published/released for a few weeks.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/reactive-streams/reactive-streams-dotnet/pull/7/files/dadf35ab71e8e40ca3ca852a2252bab85a820ffa#r65007389>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAqd4Ah24mu0v5pEC1o6O8RHRcIewWXks5qGbqmgaJpZM4IoorW>
> .
>